migrator/migrations/sqls/20200220183445-seed-up.sql

```diff
@@ -0,0 +1,78 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE reg_state AS ENUM ('unregistered', 'registered');
+
+CREATE TABLE IF NOT EXISTS Users (
+    uid uuid DEFAULT uuid_generate_v1() UNIQUE,
+    email varchar(255) NOT NULL,
+    password varchar(255) NOT NULL,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,
+    organizer boolean DEFAULT FALSE,
```

Comment:
I take it this is the admin boolean?

Reply:
Yes. We labelled it as `organizer` in the UML, but I think `admin` is better.